mm2src/derives/enum_from/src/lib.rs

```diff
@@ -84,19 +85,85 @@ pub fn enum_from_trait(input: TokenStream) -> TokenStream {
     }
 }
 
+/// `EnumFromStringify` is very useful for generating `From<T>` trait from one enum to another enum
+/// currently, this crate can only convert enum variant with only some basic inner type such as `String`, and `Enum`
+/// type just like the example below. Can not be used for tuple, struct etc for now .
+/// More support will be added.
+///
+///
+/// ### USAGE:
+/// ```rust
```

Comment:
Could you add one line less above, and one line more below `USAGE`

Reply:
done